docs/docs/schema-customization.md

```diff
@@ -111,6 +114,7 @@ type AuthorJson implements Node {
 > Start your project with `GATSBY_GRAPHQL_IDE=playground gatsby develop`, open the
 > playground at `http://localhost:8000/___graphql` and inspect the `Schema` tab on
 > the right.
+This Graphql playground provides you an interface to query on the left side and get its queried outputs on the right side of the window.
```

Comment:
This line would be better fit under the "Using the GraphQL Playground" page and a link to that page could be added to the blockquotes in lines 113-116